aggregator/src/main/scala/weco/concepts/aggregator/sources/StdInSource.scala

```diff
@@ -0,0 +1,21 @@
+package weco.concepts.aggregator.sources
+
+import akka.NotUsed
+import akka.stream.scaladsl.{Source, StreamConverters}
+import grizzled.slf4j.Logging
+import weco.concepts.common.source.Scroll
+import weco.concepts.aggregator.Main.maxFrameKiB
```

Comment:
Is this a circular dependency / should there be a `Config` singleton or case classes that get passed through as in the API etc / do we care?

Reply:
Good point about possible circular dependency.

I originally passed the framesize as a parameter, because that's the way I like it.

Then I used the IDE "refactor" to move something around, and this happened, and it worked, so I thought I should keep it that way.